sdk/samples/azidentity/example_SDKV1_test.go

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azidentitysamples
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/azure-sdk-for-go/sdk/to"
+	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2019-05-01/resources"
+
+	"github.com/jongio/azidext/go/azidext"
+)
+
+// Please note that the examples in this file are using the Azure SDK for Go V1 code base, along
+// with azidentity package from the Azure SDK for Go V2.
+// The adapter in the azidext package provides a simple way to integrate azidentity credentials
+// as authorizers for the V1 code base.
+
+const (
+	groupName = "samplegroup"
+)
+
+// Environment variables required for EnvironmentCredential to work and/or DefaultAzureCredential
+var (
+	clientID       = os.Getenv("AZURE_CLIENT_ID")
+	clientSecret   = os.Getenv("AZURE_CLIENT_SECRET")
+	subscriptionID = os.Getenv("AZURE_SUBSCRIPTION_ID")
+	tenantID       = os.Getenv("AZURE_TENANT_ID")
+)
+
+var (
+	location  = os.Getenv("AZURE_LOCATION")
+	userAgent = "azidentitysample"
+)
+
+// Example for using the DefaultAzureCredential through the NewDefaultAzureCredentialAdapter and assigning the credential to the
+// SDK V1 authorizer.
+func getGroupsClientWithDefaultAzureCredential() resources.GroupsClient {
```

Comment:
Since using the adapter with azidentity is the focus here, I'd make these examples and remove the other two.

Reply:
Done